staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -191,7 +191,7 @@ type PersistentVolumeSource struct {
 	// exposed to the pod. Provisioned by an admin.
 	// More info: https://releases.k8s.io/HEAD/examples/volumes/glusterfs/README.md
 	// +optional
-	Glusterfs *GlusterfsVolumeSource `json:"glusterfs,omitempty" protobuf:"bytes,4,opt,name=glusterfs"`
+	Glusterfs *GlusterfsPersistentVolumeSource `json:"glusterfs,omitempty" protobuf:"bytes,4,opt,name=glusterfs"`
```

Comment:
Sorry I'm late to the party. Is this a backwards compatible change?  Will clients that programatically interact with the API start failing because the type has changed?

Reply:
The type of the go field does not impact what gets serialized to the wire, so the type change does not affect wire compatibility (against the API or against etcd).

The type of the go struct field is changing. Depending on how a client made use of this object, it might have to adjust how it constructed a struct of this type. If all it did was retrieve objects from the API and access fields (e.g. `pv.Spec.Glusterfs.EndpointsName`) no change would be required.

Reply:
Ah ok, so if someone synced their go-client to the latest, then it's possible they will have to recompile, but older applications using the old clients will continue to work.  Is this something we need to call out in any release notes?

Reply:
yes, worth a release note, both for the new capability, and for the client-go impact